test/onigumo_cli_test.exs

```diff
@@ -83,6 +83,16 @@ defmodule OnigumoCLITest do
       end
     end
 
+    test("Usage message ends with exactly one newline") do
+      output = capture_io(fn -> Onigumo.CLI.main([]) end)
+      assert String.match?(output, ~r/\N\n\z/)
```

Comment:
`String.match?(output, ~r/\N\n\z/)` is rather cryptic. What about a private method like `ends_with_exactly_one_newline?`?